AlphaWallet/Tokens/Collectibles/ViewModels/NFTCollectionInfoPageViewModel.swift

```diff
@@ -81,7 +81,7 @@ final class NFTCollectionInfoPageViewModel {
             .flatMapLatest { [nftProvider] tokenHolder -> AnyPublisher<Loadable<Stats, PromiseError>, Never> in
                 if let collectionId = tokenHolder?.values.collectionId, collectionId.trimmed.nonEmpty {
                     return nftProvider.collectionStats(collectionId: collectionId)
-                        .publisher()
+                        .receive(on: RunLoop.main)
```

Comment:
Shouldn't upstream be already on `.main` by convention? If it isn't, can we make it so in a separate PR?